lib/src/controls/form/text_box.dart

```diff
@@ -686,7 +688,7 @@ class _TextBoxState extends State<TextBox>
       child: IgnorePointer(
         ignoring: !enabled,
         child: AnimatedContainer(
-          duration: theme.fasterAnimationDuration,
+          duration: const Duration(milliseconds: 0),
```

Comment:
Why disabling the animation?

Reply:
If the animation is on, then when you click on the field, a flashing in a different color occurs. I looked at the XAML Controls Gallery and did not notice the animation for this action there, so I disabled it.

Reply:
I don’t decide if we disable animation or not (@bdlukaa take the decision), but if you remove the animation, don’t put a duration of 0 and remove totally the AnimatedContainer